src/uniprotkb/utils/resultsUtils.ts

```diff
@@ -15,16 +15,19 @@ const facetsAsArray = (facetString: string): SelectedFacet[] => {
     };
   });
 };
+
 export type URLResultParams = {
   query: string;
   selectedFacets: SelectedFacet[];
   sortColumn: SortableColumn;
   sortDirection: SortDirection;
   activeFacet?: string;
+  direct?: boolean;
```

Comment:
We need to think about a way to redirect to an entry directly if there's only 1 result - IDK if the current website uses `direct` to redirect to it or not?

Reply:
For a normal search on the current website I don't know, because there's no frontend redirect, it's probably handled by the server directly.
We need to be cautious here and we need to have a UX discussion of this behaviour if we are to apply it by default, to reasons on the top of my head:
 - What do you do if you only have 1 result but it's just because it's using a really restrictive query or facets?
   - We need to redirect only if the query was an accession? or an ID? or part of an ID (uniparc search for "123" => autoredirect to UPI00...000123)?
   - We need to be clearer when a search result page is a search result page and when we have an entry page (thinking UniRef here) because if I get lost sometimes and don't realise on which type of page I am, I can't expect a casual user to understand that they've been redirected "for their good".

Also, not a fan of having different behaviour (redirect or not) on 2 similar buttons really next to each other ("expand" and "list" clusters). But I'm just matching current behaviour in this PR for now.